example/ios/Runner/Info.plist

```diff
@@ -45,5 +45,9 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSBluetoothAlwaysUsageDescription</key>
+    <string>The app uses bluetooth to find, connect and transfer data between different devices</string>
+    <key>NSBluetoothPeripheralUsageDescription</key>
+    <string>The app uses bluetooth to find, connect and transfer data between different devices</string>
```

Comment:
Please fix the indentation (looks like you edited the file manually).

Reply:
Done